lib/spack/spack/test/cmd/create.py

```diff
@@ -16,6 +16,15 @@
 create = SpackCommand('create')
 
 
+def _validate_url(url):
+    return
+
+
+@pytest.fixture(autouse=True)
+def url_check(monkeypatch):
+    monkeypatch.setattr(spack.util.url, 'require_url_format', _validate_url)
```

Comment:
The rebase turned off the auto merge, I'd merge it myself but I'm personally curious whether these tests can be made to pass without this monkeypatching. IIRC we spoke about a few cases like this in https://github.com/spack/spack/pull/27021 and my general goal was to not use it where possible. I notice that @alalazo has already approved this, so feel free to merge it if this is not a concern